src/layout/DashboardLayout/Sidebar/index.tsx

```diff
@@ -4,21 +4,19 @@ import ListItemButton from '@mui/material/ListItemButton';
 import ListItemText from '@mui/material/ListItemText';
 import Grid from '@mui/material/Grid';
 import { useNavigate } from 'react-router-dom';
+import useCurrentUser from '@/hooks/useCurrentUser';
 
 export const mainListItems = <React.Fragment></React.Fragment>;
 
 const SideBar = () => {
   const navigate = useNavigate();
-  return (
-    <List component="nav" sx={{ backgroundColor: '#ED81AD', height: '100vh' }}>
-      <Grid
-        container
-        direction="column"
-        justifyContent="center"
-        alignItems="center"
-        sx={{ paddingTop: 8 }}
-      >
-        <Grid item xs={6}>
+  const { currentUserError, data, currentUserLoading } = useCurrentUser();
+  const role = data?.data.data.getCurrentUser.role;
```

Comment:
```suggestion
  const role = data?.data.data.getCurrentUser.role;
```
  const role = data?.data?.data?.getCurrentUser?.role;